lib/blacklight/configuration.rb

```diff
@@ -78,6 +78,10 @@ def header_component
     # @since v5.2.0
     # @return [String] The url path (relative to the solr base url) to use when requesting only a single document
     property :document_solr_path, default: 'get'
+    # @!attribute json_solr_path
+    # @since v7.34.0
+    # @return [String] The url path (relative to the solr base url) to use when using Solr's JSON Query DSL (as with the advanced search)
+    property :json_solr_path, default: 'advanced'
```

Comment:
```suggestion
    property :json_solr_path, default: nil
```

Should we set this to `nil` by default for backwards compatibility (esp. for people who worked around that upstream solr bug some other way)?